pkgs/tools/misc/expect/default.nix

```diff
@@ -9,19 +9,27 @@ stdenv.mkDerivation rec {
     sha256 = "0d1cp5hggjl93xwc8h1y6adbnrvpkk0ywkd00inz9ndxn21xm9s9";
   };
 
-  buildInputs = [ tcl ];
-  nativeBuildInputs = [ makeWrapper autoreconfHook ];
-
-  hardeningDisable = [ "format" ];
+  patches = [
+    (fetchpatch {
+      url = "https://raw.githubusercontent.com/buildroot/buildroot/master/package/expect/0001-enable-cross-compilation.patch";
```

Comment:
please fetch this patch from an explicit commit, not master

Reply:
Yes otherwise checksums break if the patch gets modified/removed.